src/snyk/common/views/webviewProvider.ts

```diff
@@ -21,11 +24,23 @@ export abstract class WebviewProvider {
 
   protected onPanelDispose(): void {
     this.panel = undefined;
+
+    while (this.disposables.length) {
```

Comment:
I'm not really sure, how this.disposables is populated.

Reply:
VS Code API adds the disposable for us to this array. See the last argument provided for `this.panel.onDidChangeViewState()` function in `src/snyk/snykCode/views/suggestion/codeSuggestionWebviewProvider.ts`.

Here's the API description: https://code.visualstudio.com/api/references/vscode-api#Event%3CT%3E